lib/shoulda/matchers/independent/delegate_method_matcher.rb

```diff
@@ -327,8 +338,21 @@ def ensure_delegate_object_has_been_specified!
           end
         end
 
+        def subject_delegates_to_nil_delegate_object_correctly?
+          return true unless delegate_object_is_nil_allowed
```

Comment:
I need to return true when `delegate_object_is_nil_allowed ` was not set. If I do what you say I would have to add an else block returning true, it would look like:
```ruby
def subject_delegates_to_nil_delegate_object_correctly?
  if delegate_object_is_nil_allowed
    # My code here...
  else
    true
  end
end
```

I think it looks ugly, what do you think?